pandas/tests/io/test_sql.py

```diff
@@ -2698,6 +2700,20 @@ def test_drop_table(conn, request):
         assert not insp.has_table("temp_frame")
 
 
+@pytest.mark.parametrize("conn", sqlalchemy_connectable + ["sqlite_buildin"])
```

Comment:
Can you use the `all_connectable` here? We should have it work for ADBC as well

Reply:
Hey o/. Thanks a ton for the fast review.
I'm unit testing the method `delete_rows` here. I didn't implement this method for the `ADBCDatabase` following the existing convention.

Do you believe we should implement `delete_rows` for the `ADBCDatabase` ?

Do you see value in unit testing `delete_rows` as its functionality is tested at `test_to_sql_exist` ?

Reply:
Yea generally we strive to have the same behavior across all drivers; we should only really pare back from using `all_connectable` for very specific use cases

I think the fixture in `test_to_sql_exist` is a good start, but it is missing assertions around functionality specific to `delete_rows`; that test would pass if you re-used the same behavior of "replace", but that's not what we are after

Reply:
Perfect. We're 100% aligned then. That's why I created the unit test.

I'm going to refactor it so the ADBC backend implements this new functionality

Reply:
Note that ADBC doesn't use SQLAlchemy so I think will have to write the SQL statement. I think that's OK given `DELETE FROM` is ANSI-SQL, but will also want to be careful that we aren't at risk of SQL injection

Reply:
I don't think a prepared statement is the right approach; I believe that syntax you are using is also specific to SQLite

I think you can just use f-strings. Ultimately, the driver should be preventing against SQL injection, so as long as we test that is the case I don't think we need any special handling on our end

Reply:
Prepared statements do not bind table names, which is the issue you are running into. 

> Humm... I'm not sure about this statement. AFAIK there's no driver preventing out of box protection of SQL Injection. This responsibility lies to whom is using it, and we're not protected as f-strings are used all over the ADBC backend. Might be worth creating a ticket to check this ?

The presence of f-strings are a red herring, as the ADBC drivers are implemented in lower level languages. Upstream, there are PRs like:

https://github.com/apache/arrow-adbc/issues/643

and 

https://github.com/apache/arrow-adbc/issues/1340

that attempt to guard against injection. Are you able to circumvent those safeguards?

Reply:
Today I learned 🙇 . Thanks a lot - for real. I'll submit a patch in a few.